radio/src/gui/colorlcd/layouts/topbar_impl.cpp

```diff
@@ -32,6 +32,9 @@ TopbarImpl::TopbarImpl(Window * parent) :
 
 unsigned int TopbarImpl::getZonesCount() const
 {
+#if defined(TELEMETRY_MAVLINK) && defined(INTERNAL_GPS)
```

Comment:
Shouldn't it be just `INTERNAL_GPS`? As I understand, this is for all GPS users, right?

Reply:
pl read intros of PR's, sometimes something relevant is said there :)

> As is, it is an addition to the MAVLink for EdgeTx PR. However, if this is generally liked by users (which might be the case), it could be also made available to all users by simply removing the dependency on TELEMETRY_MAVLINK in the defines. Just say so :)

Reply:
@olliw42 I did, but I would love to have it even if MAVlink is not activated :-)

Reply:
Ok so "I just say so" :-D

Reply:
Thx a lot!

Reply:
that's language I understand 🤣 so I will do so... 
it's actually cool this is receibed so positively, I wasn't sure...